website/docs/intro/migration/terraform-1.5-or-lower.mdx

```diff
@@ -1,119 +1,131 @@
+sure you have an up to date and *tested* disaster recovery plan.
+
+## Step 1: Upgrading Terraform
+
+This migration guide is only valid for Terraform 1.5.7. If you are on a Terraform version below 1.5.7, please upgrade
+to at least Terraform version 1.5.7 before proceeding with the migration by following the
+[Terraform upgrade guide](https://developer.hashicorp.com/terraform/language/v1.5.x/upgrade-guides). If you are on a
+higher Terraform version, please select the appropriate migration guide for your Terraform version.
+
+## Step 2: Apply all changes with Terraform
+
+Before proceeding, make sure that you apply all changes with `terraform apply`. Running `terraform plan` should result
+in no planned changes. While you can switch to OpenTofu with pending changes, it is not recommended.
+
+```
+$ terraform plan
+
+...
+
+No changes. Your infrastructure matches the configuration.
+
+Terraform has compared your real infrastructure against your
+configuration and found no differences, so no changes are needed.
+```
+
+## Step 3: Install OpenTofu
+
+As a first step, please [follow the installation instructions for the OpenTofu CLI tool](install/index.mdx). Make sure
+to install OpenTofu 1.6.2. Please test if you can successfully execute the `tofu` command and that it returns the
+correct version:
+
+```
+$ tofu --version
+OpenTofu v1.6.2
+on linux_amd64
+```
+
+## Step 4: Back up your state file
+
+Before you begin using the `tofu` binary on your Terraform code, make sure to back up your state file. If you are using
+a local state file, you can simply make a copy of your `terraform.tfstate` file in your project directory.
+
+If you are using a remote backend such as an S3 bucket, make sure that you follow the backup procedures for the
+backend and that you exercise the restore procedure at least once.
+
+## Step 5: Initialize OpenTofu
+
+:::warning
+
+Should any of the following steps fail, please do not proceed and follow the
+[rollback instructions below](#rolling-back-to-terraform-and-reporting-issues) instead.
+If you suspect the failure may be the result of a bug in OpenTofu,
+[please help us by opening an issue](https://github.com/opentofu/opentofu/issues).
+
+:::
+
+Now you are ready to migrate. Run `tofu init` in the directory where your Terraform code resides. OpenTofu will download
+any providers and modules referenced in your configuration from the OpenTofu registry.
+
+## Step 6: Inspect the plan
+
+Once initialized, run `tofu plan` and ensure that there are no pending changes similar to step 1 above. If there are
+unexpected changes in the plan, roll back to Terraform and troubleshoot your migration. (See the Troubleshooting section
+below.)
+
+```
+$ tofu plan
+
+...
+
+No changes. Your infrastructure matches the configuration.
+
+OpenTofu has compared your real infrastructure against your
+configuration and found no differences, so no changes are needed.
+```
+
```

Comment:
I would suggest also doing an empty tofu apply after the tofu plan, as that rewrites the statefile.

It's not strictly necessary, but safer.

Reply:
Really, I would replace step 7 with that - empty apply.